man/systemd.net-naming-scheme.xml

```diff
@@ -525,6 +525,15 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><constant>v256</constant></term>
+
+          <listitem><para>Added support for ethernet devicetree aliases pointing to ethernet-ports switch.</para>
```

Comment:
I think it makes more sense to say something like

> Added support for ethernet devicetree aliases pointing to switch ports.

Reply:
Fixed, thank you